### PR TITLE
Add cwl-upgrader

### PR DESCRIPTION
**Pull request by kinow**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Unblocks https://github.com/conda-forge/staged-recipes/pull/19969
Related to https://github.com/common-workflow-language/cwl-utils/issues/138

### CODE REVIEW

**Comment by kinow** [comment]
@mr-c whenever you have time can you just reply here saying that you are OK being a maintainer of this recipe (pull request requirement). Thanks!

**Comment by mr-c** [comment]
I'm happy to be a maintainer of this package, yep !

**Comment by kinow** [comment]
@conda-forge/help-python, ready for review!

**Comment by kinow** [comment]
This PR was created with greyskull, following the docs from https://conda-forge.org/docs/maintainer/adding_pkgs.html

**Review by carterbox — CHANGES_REQUESTED** [review]
Please convert this recipe to noarch: python? Suggestion how below.

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

File: recipes/cwl-upgrader/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cwl-upgrader-{{ version }}.tar.gz
+  sha256: 3372b87af6ab6154716462f97561620eecd786798a68a2a64a98529340f52e1f
+
+build:
+  skip: true  # [py>=400 or win]
```
Comment (carterbox) [inline recipes/cwl-upgrader/meta.yaml:13]: There's a minimum python version upstream. python>=3.6

**Review by carterbox — CHANGES_REQUESTED** [review]

File: recipes/cwl-upgrader/meta.yaml (hunk 1 above)
Comment (mr-c) [reply recipes/cwl-upgrader/meta.yaml:13]:
```suggestion
```

**Comment by kinow** [comment]
Thanks for the review @carterbox & @mr-c ! Applied suggested fixes, and then fixed the linter issue. Should be ready for review again :tada:

**Review by mr-c — APPROVED** [review]

**Comment by mr-c** [comment]
Dear @conda-forge/staged-recipes ; can we get a re-review? `cwl-upgrader` is needed by https://github.com/conda-forge/staged-recipes/pull/19969 (`cwl-utils`) which is then required for the new version of `cwltool` https://github.com/conda-forge/cwltool-feedstock/pull/112

Thanks!

**Comment by mr-c** [comment]
Thank you @carterbox 🤓

**Pull request merged**